static/src/stylesheets/module/content-garnett/_types.scss

```diff
@@ -1,4 +1,197 @@
+$quote-tail: 25px;
+// ********** Article Type **********
 .content--type-article {
+    .content__article-body {
+        @include mq(desktop) {
+            border-left: solid 1px $neutral-5;
+            padding-left: $gs-gutter/2;
+            margin-left: 0;
+        }
+        @include mq(leftCol) {
+            margin-left: -$gs-gutter/2;
+        }
+    }
+
+    .badge,
+    .content__meta-container {
+        @include mq(leftCol) {
+            width: gs-span(2) -$gs-gutter/2;
+        }
+        @include mq(wide) {
+            width: gs-span(3) -$gs-gutter/2;
+        }
+    }
+    // Quote base styles
+    .element-pullquote {
+        background-color: $neutral-garett-1;
+        border: solid 1px $neutral-5;
+        margin: $gs-gutter/4 0 $gs-baseline*2;
+        padding: $gs-baseline/2 $gs-gutter/2 $gs-baseline;
+        position: relative;
+        clear: none;
+        float: none;
+        width: inherit;
+        @include mq(mobile) {
+            margin-left: -$gs-gutter/2 - 1px;
+        }
+        @include mq(mobileLandscape) {
+            margin-right: $gs-gutter/2;
+            margin-left: -$gs-gutter;
+        }
+        @include mq(leftCol) {
+            margin-left: -$gs-baseline*2;
+        }
+        // Quote tail
+        &:after {
+            @include mq(mobile) {
+                content: '';
+                width: $quote-tail;
+                height: $quote-tail;
+                border-radius: 0 0 $quote-tail;
+                position: absolute;
+                bottom: -$quote-tail - 2px;
+                left: $gs-gutter/2;
+                background-color: $neutral-garett-1;
+                border: solid 1px $neutral-5;
+                border-top: solid 1px $neutral-garett-1;
+            }
+            @include mq(mobileLandscape) {
+                left: $gs-gutter;
+            }
+            @include mq(desktop) {
+                left: $gs-gutter - $gs-baseline;
+            }
+            @include mq(leftCol) {
+                left: $gs-baseline;
+            }
+        }
+
+        .pullquote-cite,
+        .pullquote-paragraph {
+            display: inline;
+            font-weight: 100!important;
```

Comment:
How come we are using !important? Is this temporary?

Reply:
yeah can we not?

Reply:
sorry - as in, is it possible to avoid it?

Reply:
Yup is fixed, changed the weight in the mixin